### PR TITLE
[FE-3396] Create brand

### PR DESCRIPTION
**Pull request by xixe**

### CODE REVIEW

File: src/modules/brand/components/UploadAvatarField/components/InnerBlock/innerBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```
Comment (BigBadAlien) [inline src/modules/brand/components/UploadAvatarField/components/InnerBlock/innerBlock.tsx:1]: innerBlock.tsx -> InnerBlock.tsx

File: src/modules/brand/components/UploadAvatarField/components/InnerBlock/innerBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import { t } from 'modules/i18n/utils/intl';
+import { PencilIcon } from 'modules/common/components/Icons/PencilIcon';
+import { useUploadAvatarFieldStyles } from '../../useUploadAvatarFieldStyles';
+
+export interface IInitialBlock {
+  avatar: string;
+  value: any;
+  input: JSX.Element;
+  inputRef: any;
+  handleReset: any;
```
Comment (BigBadAlien) [inline src/modules/brand/components/UploadAvatarField/components/InnerBlock/innerBlock.tsx:17]: inputRef: React.MutableRefObject<HTMLInputElement | null> // likelly
handleReset: () => void;

File: src/modules/form/components/UploadFileField/UploadFileField.tsx [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+const MAX_SIZE: Bytes = 31457280;
+
+interface IFieldProps extends FieldRenderProps<string> {
+  maxSize?: Bytes;
+  className?: string;
+  acceptsHint?: string[];
+  accepts?: string[];
+  fitView?: boolean;
+  label?: string;
+  InitialBlockComponent?: any;
```
Comment (BigBadAlien) [inline src/modules/form/components/UploadFileField/UploadFileField.tsx:30]: any -> `ReactElement<props>`

File: src/modules/form/components/UploadFileField/useUploadFileStyles.ts [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      height: '100%',
+    },
+    '& $previewUploadedImage': {
+      height: '100%',
+      backgroundPosition: 'center',
+      backgroundSize: 'cover',
+      backgroundRepeat: 'no-repeat',
+    },
+    '& $previewUploadedImageFit': {
+      backgroundSize: 'contain !important',
```
Comment (BigBadAlien) [inline src/modules/form/components/UploadFileField/useUploadFileStyles.ts:49]:
```
'& $previewUploadedImageFit': {
   '&&': {
            backgroundSize: 'contain',
   }
}
```

File: src/modules/brand/components/UploadAvatarField/components/InnerBlock/innerBlock.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  Typography,
+} from '@material-ui/core';
+import { Button } from 'modules/uiKit/Button';
+import { t } from 'modules/i18n/utils/intl';
+import { PencilIcon } from 'modules/common/components/Icons/PencilIcon';
+import { useUploadAvatarFieldStyles } from '../../useUploadAvatarFieldStyles';
+
+export interface IInitialBlock {
+  avatar: string;
+  value: any;
```
Comment (BigBadAlien) [inline src/modules/brand/components/UploadAvatarField/components/InnerBlock/innerBlock.tsx:14]:
```
  value?: File;
```

No?

**Review by BigBadAlien — APPROVED** [review]

File: src/modules/brand/components/UploadAvatarField/components/InnerBlock/innerBlock.tsx (hunk 5 above)
Comment (xixe) [reply src/modules/brand/components/UploadAvatarField/components/InnerBlock/innerBlock.tsx]: string

File: src/modules/form/components/UploadFileField/useUploadFileStyles.ts (hunk 4 above)
Comment (xixe) [reply src/modules/form/components/UploadFileField/useUploadFileStyles.ts]: Removed !important
```
'& $previewUploadedImage': {
      ....
      backgroundRepeat: 'no-repeat',
      '&$previewUploadedImageFit': {
        backgroundSize: 'contain',
      },
    },
```

File: src/modules/form/components/UploadFileField/UploadFileField.tsx (hunk 3 above)
Comment (xixe) [reply src/modules/form/components/UploadFileField/UploadFileField.tsx]: Replaced to JSX.Element

**Pull request merged**